### PR TITLE
Add Dockerfile

### LINKED ISSUE #11: Need DockerFile
cc @mostlygeek

### PR DESCRIPTION
**Pull request by oremj**
Fixes #11

### CODE REVIEW

**Comment by oremj** [comment]
@mostlygeek r?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.5
+
+ENV PROJECT github.com/mozilla-services/go-bouncer
+
+COPY . /go/src/$PROJECT
+
+ENV GOPATH /go/src/$PROJECT/Godeps/_workspace:$GOPATH
```
Comment (mostlygeek) [inline Dockerfile]: Put this in line 4. Each instruction in a Dockerfile creates a new layer. Moving it to line 4 will be more cache friendly.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+RUN go install $PROJECT && go install $PROJECT/go-sentry
```
Comment (mostlygeek) [inline Dockerfile]: Will `golang:1.5` automatically run Godeps?

Comment (oremj) [inline Dockerfile]: I don't need godeps to run. The deps will already be in the repository.

**Comment by mostlygeek** [comment]
R+

**Pull request merged**